packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -359,15 +359,16 @@ abstract class Route<T> {
 
   /// Discards any resources used by the object.
   ///
+  /// This method does not remove its overlay entries from the overlay. The
```

Comment:
Just say: This method should not remove its [overlayEntries] from the [Overlay]. The [Navigator] is in charge of doing that.

Reply:
The comment on line 369 makes me wonder if we should strictly says The [Navigator]? 
I would like to rephrase as follow
This method should not remove its [overlayEntries] from the [Overlay]. The object's owner is in charge of doing that.
let me know if you still think we should narrow down to navigator